activesupport/test/core_ext/time_with_zone_test.rb

```diff
@@ -421,11 +421,29 @@ def test_time_at
     assert_equal time, Time.at(time)
   end
 
-  def test_to_time
-    with_env_tz "US/Eastern" do
-      assert_equal Time, @twz.to_time.class
-      assert_equal Time.local(1999, 12, 31, 19), @twz.to_time
-      assert_equal Time.local(1999, 12, 31, 19).utc_offset, @twz.to_time.utc_offset
+  def test_to_time_with_preserve_timezone
```

Comment:
These tests now check the `preserve_timezone` which is included into TWZ as part of `DateAndTime::Compatibility` included at the top.